sdk/evm/tbrv3/contract.ts

```diff
@@ -82,7 +86,18 @@ export interface TbrPartialTx {
 }
 
 
-export type RelayingFeesInput = LayoutToType<typeof relayingFeesInputLayout>;
+export interface RelayingFeesInput {
+  /**
+   * Token addresses involved in the transfers.
+   * Must be hex encoded and '0x' prefixed.
+   */
+  readonly tokens: readonly string[];
```

Comment:
That's why the good lord gave us `EvmAddress` :P.
General rule: Don't say in a comment what you can enforce with a type.

Reply:
lol, we can definitely implement that. I was meaning to since the price oracle sdk already uses it.

Reply:
Btw, I replaced all `string` types for addresses already. There are some address uses as keys so those are kept as `string`s though.